libvirt/README.md

```diff
@@ -44,3 +44,222 @@ If you want to set default password for podvm debugging then you can use guestfi
+
+```
+$ kubectl get pods -n confidential-containers-system
+NAME                                              READY   STATUS    RESTARTS   AGE
+cc-operator-controller-manager-5df7584679-5dbmr   2/2     Running   0          3m58s
+cloud-api-adaptor-daemonset-libvirt-vgj2s         1/1     Running   0          3m57s
+$ kubectl logs pod/cloud-api-adaptor-daemonset-libvirt-vgj2s -n confidential-containers-system
++ exec cloud-api-adaptor-libvirt libvirt -uri 'qemu+ssh://wmoschet@192.168.1.107/system?no_verify=1' -data-dir /opt/data-dir -pods-dir /run/peerpod/pods -network-name default -pool-name default -socket /run/peerpod/hypervisor.sock
+2022/11/09 18:18:00 [helper/hypervisor] hypervisor config {/run/peerpod/hypervisor.sock  k8s.gcr.io/pause:3.7 /run/peerpod/pods libvirt}
+2022/11/09 18:18:00 [helper/hypervisor] cloud config {qemu+ssh://wmoschet@192.168.1.107/system?no_verify=1 default default /opt/data-dir}
+2022/11/09 18:18:00 [helper/hypervisor] service config &{qemu+ssh://wmoschet@192.168.1.107/system?no_verify=1 default default /opt/data-dir}
+```
+
+You will also notice that Kubernetes [*runtimeClass*](https://kubernetes.io/docs/concepts/containers/runtime-class/) resources
+were created on the cluster, as for example:
+
+```
+$ kubectl get runtimeclass
+NAME        HANDLER     AGE
+kata        kata        7m41s
+kata-clh    kata-clh    7m41s
+kata-qemu   kata-qemu   7m41s
+```
+
+## Create a sample peer-pods pod
+
+At this point everything should be fine to get a sample Pod created. Let's first list the running VMs so that we can later check
+the Pod VM will be really running. Notice below that we got only the cluster node VMs up:
+
+```
+$ virsh -c qemu:///system list
+ Id   Name                 State
+------------------------------------
+ 3    peer-pods-master-0   running
+ 4    peer-pods-worker-0   running
+```
+
+Create the *sample_busybox.yaml* file with the following content:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    run: busybox
+  name: busybox
+spec:
+  containers:
+  - image: quay.io/prometheus/busybox
+    name: busybox
+    resources: {}
+  dnsPolicy: ClusterFirst
+  restartPolicy: Never
+  runtimeClassName: kata
+```
+
+And create the Pod:
+
+```
+$ kubectl apply -f sample_busybox.yaml
+pod/busybox created
+$ kubectl wait --for=condition=Ready pod/busybox
+pod/busybox condition met
+```
+
+Check that the Pod VM is up and running. See on the following listing that *podvm-busybox-88a70031* was
+created:
+
+```
+$ virsh -c qemu:///system list
+ Id   Name                     State
+----------------------------------------
+ 5    peer-pods-master-0       running
+ 6    peer-pods-worker-0       running
+ 7    podvm-busybox-88a70031   running
+```
+
+You should also check that the container is running fine. For example, compare the kernels are different as shown below:
+
+```
+$ uname -r
+5.17.12-100.fc34.x86_64
+$ kubectl exec pod/busybox -- uname -r
+5.4.0-131-generic
+```
```

Comment:
I wonder if a clean-up/uninstall might be useful? That can be another PR though.

Reply:
Done.

Reply:
Sorry, I wasn't particularly clear here. I was more thinking about the operator install, in case a user wanted to update (which I think would be an uninstall and then re-install, but if installing over the top works then this is fine)?

Reply:
ah, ok. I never installed a new version over the top, so no idea it works (@bpradipt ?). If so then it is a good use test case for the operator tests.

I will add some words about uninstall/install the operator.

Reply:
Done